python-client/src/main/python/janusgraph_grpc_python/management/management_pb2_grpc.py

```diff
@@ -0,0 +1,664 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
You should exclude auto generate code.

Reply:
Oh, then pass the instructions for building later on?
My first priority was to just make it work so yea can be changed :-) Later on there will be install script to do things (Like we have to edit the imports here)